test/kubemark/start-kubemark.sh

```diff
@@ -167,6 +139,35 @@ EOF
   echo "Kubeconfig file for kubemark master written to ${LOCAL_KUBECONFIG}."
 }
 
+# Copy all the necessary resource files (scripts/configs/manifests) to the master.
```

Comment:
What is the point of changing the order of functions? You can just change the order in which you call them...